src/main/java/org/mskcc/cmo/messaging/model/PatientToPatientEntity.java

```diff
@@ -0,0 +1,51 @@
+package org.mskcc.cmo.messaging.model;
+
+import java.io.Serializable;
+import java.util.Collection;
+import org.neo4j.ogm.annotation.EndNode;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.RelationshipEntity;
+import org.neo4j.ogm.annotation.StartNode;
+
+/**
+ *
+ * @author ochoaa
+ */
+@RelationshipEntity(type="PX_TO_PX")
+public class PatientToPatientEntity implements Serializable {
+    @Id @GeneratedValue
+    private Long id;
+    private Collection<String> linkedPatientNames;
+    @StartNode
+    private LinkedPatient linkedPatient;
+    @EndNode
+    private PatientMetadata patientMetadata;
+
+    public PatientToPatientEntity() {}
+
+    public Collection<String> getLinkedPatientNames() {
```

Comment:
This class represents an edge between patient nodes.  Will the size of LinkedPatientNames ever be more than 2?  How do you know which is the source and target nodes?

Reply:
the relationship has a direction, at least in our example  graph db so the start node for this patient-to-patient entity will always have to be a `LinkedPatient` node and the end node will be the `PatientMetadata` it's linked to